index.html

```diff
@@ -21,6 +21,17 @@ <h2>
       </button>
     </form>
 
+    <form action="/messages" method="delete">
+      <p>
+        Name: <input type="text" name="from" placeholder="Your Name" /> <br />
+        Message: <input type="text" name="text" placeholder="The message..." />
+        <br />
+      </p>
+      <button type="submit">
+        Delete
+      </button>
+    </form>
```

Comment:
This doesn't connect to the delete method you have written in server.js. The reason it doesn't work is because you need to provide the id as a path parameter (as set out in your API URL here: https://github.com/CodeYourFuture/node-challenge-chat-server/pull/138/files#diff-a4c65ede64197e1a112899a68bf994485b889c4b143198bac4af53425b38406fR70).

Reply:
From an html form, it's not really possible to append path parameters like your api requires (because your endpoint looks like /messages/:id....) The only way you can do this from a vanilla html/javascript application is to: 
1) change your form to have id as an input (because you need to tell the api which message id you want to delete. 
2) create a javascript handler for the form which takes the input of the form, forms the URL appropriately and then submits a delete request. 

To be honest, this is beyond the scope of this particular exercise and is much easier to do in react. For the purposes of this merge request, it is good enough that your delete method works in postman (which it does), so well done!

Reply:
From an html form, it's not really possible to append path parameters like your api requires (because your endpoint looks like /messages/:id....) The only way you can do this from a vanilla html/javascript application is to: 
1) change your form to have id as an input (because you need to tell the api which message id you want to delete. 
2) create a javascript handler for the form which takes the input of the form, forms the URL appropriately and then submits a delete request. 

To be honest, this is beyond the scope of this particular exercise and is much easier to do in react. For the purposes of this merge request, it is good enough that your delete method works in postman (which it does), so well done!